src/Http/Request.php

```diff
@@ -110,58 +71,42 @@ public function __construct(array $query = [], array $request = [], array $cooki
      */
     public static function capture()
     {
-        $request = new static($_GET, $_POST, $_COOKIE, $_FILES, $_SERVER);
-
-        if (
-            Str::startsWith($request->header('CONTENT_TYPE', ''), 'application/x-www-form-urlencoded') &&
-            in_array($request->realMethod(), ['PUT', 'PATCH', 'DELETE'])
-        ) {
-            parse_str($request->getContent(), $data);
-            $request->request = $data;
-        }
-
-        $request->request = $request->getInputSource();
-
-        return $request;
+        return new static($_GET, $_POST, $_COOKIE, $_FILES, $_SERVER);
     }
 
     /**
      * Create a new request from this one
      *
      * @param self $from
-     * @param \Radiate\Http\Request|null $to
+     * @param \WP_REST_Request|null $to
      * @return \Radiate\Http\Request
      */
-    public static function createFrom(self $from, ?Request $to = null)
+    public static function createFrom(WP_REST_Request $from, ?Request $to = null)
     {
         $request = $to ?: new static;
 
-        $request->query = $from->query;
-        $request->request = $from->request;
-        $request->cookies = $from->cookies;
-        $request->files = $from->files;
-        $request->server = $from->server;
-        $request->headers = $from->headers;
-        $request->content = $from->content;
-        $request->json = $from->json;
-
-        $request->setUserResolver($from->getUserResolver());
-        $request->setRouteResolver($from->getRouteResolver());
+        $request->set_method($from->get_method());
+        $request->set_route($from->get_route());
+        $request->set_query_params($from->get_query_params());
+        $request->set_body_params($from->get_body_params());
+        $request->set_file_params($from->get_file_params());
+        $request->set_headers($from->get_headers());
+        $request->set_body($from->get_body());
+        $request->set_default_params($from->get_default_params());
+        $request->set_attributes($from->get_attributes());
+        $request->set_url_params($from->get_url_params());
+
+        if ($from instanceof Request) {
+            $request->set_cookie_params($from->get_cookie_params());
+            $request->set_server_params($from->get_server_params());
+
+            $request->setUserResolver($from->getUserResolver());
+            $request->setRouteResolver($from->getRouteResolver());
```

Comment:
We no longer return a `Route` object so there is no need for the route resolvers